cmd/swctl/app/cmd_dependency.go

```diff
@@ -255,10 +312,15 @@ func (*Dependencies) ResizeHugePages(cli Cli, size uint) error {
 	if err != nil {
 		return err
 	}
+	allocatedHP, _ := AllocatedHugePages(cli)
+	if size != uint(allocatedHP) {
+		return errors.New("failed to allocate enough continuous memory")
```

Comment:
Add info about how much you wanted to allocate and how much you successfully allocated. This could give user some hints about how to correct its VM/bare metal setup to have enough memory.

PS: if you provide only hugepages counts, consider also adding info about how much it is in memory in MB